test/powershell/Modules/ThreadJob/ThreadJob.Tests.ps1

```diff
@@ -0,0 +1,345 @@
+﻿# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe 'Basic ThreadJob Tests' -Tags 'CI' {
+
+    BeforeAll {
+
+        $scriptFilePath1 = Join-Path $testdrive "TestThreadJobFile1.ps1"
+        @'
+        for ($i=0; $i -lt 10; $i++)
+        {
+            Write-Output "Hello $i"
+        }
+'@ > $scriptFilePath1
+
+        $scriptFilePath2 = Join-Path $testdrive "TestThreadJobFile2.ps1"
+        @'
+        param ($arg1, $arg2)
+        Write-Output $arg1
+        Write-Output $arg2
+'@ > $scriptFilePath2
+
+        $scriptFilePath3 = Join-Path $testdrive "TestThreadJobFile3.ps1"
+        @'
+        $input | foreach {
+            Write-Output $_
+        }
+'@ > $scriptFilePath3
+
+        $scriptFilePath4 = Join-Path $testdrive "TestThreadJobFile4.ps1"
+        @'
+        Write-Output $using:Var1
+        Write-Output $($using:Array1)[2]
+        Write-Output @(,$using:Array1)
+'@ > $scriptFilePath4
+
+        $scriptFilePath5 = Join-Path $testdrive "TestThreadJobFile5.ps1"
+        @'
+        param ([string]$param1)
+        Write-Output "$param1 $using:Var1 $using:Var2"
+'@ > $scriptFilePath5
+    }
+
+    AfterEach {
+
+        Get-Job | where PSJobTypeName -eq "ThreadJob" | Remove-Job -Force
+    }
+
+    It 'ThreadJob with ScriptBlock' {
+    
+        $job = Start-ThreadJob -ScriptBlock { "Hello" }
+        $results = $job | Receive-Job -Wait
+        $results | Should be "Hello"
```

Comment:
Can we use Pester 4 syntax? 
```PowerShell
$results | Should -Be "Hello"
```

Reply:
The build only pulls in Pester 3.3.9 right now which does not support this syntax.  We should wait to update tests until the default pester version supports the new syntax.